### ISSUE TITLE
Syntax Highlighting

### ISSUE THREAD

**Issue by aqce** [lhmouse/nano-win#26]
I've seen that Nano supports syntax highlighting by default, but when I try opening a simple `.py` file with nano, no syntax highlighting is applied.

**Comment by garoto** [comment]
Add a line `include "C:/my/nano/rcs/*.nanorc"` to your `.nanorc` file and then extract all the syntax highlight `.nanorc` files included in the .zip to the include dir mentioned above.

**Comment by aqce** [comment]
I think I understand what you're saying, but before I make these changes, was I supposed to install nano-win into a certain directory?

**Comment by lhmouse** [comment]
No. But wherever you do, make sure your '.nanorc' contains an absolute path to default syntax files.

**Comment by eabase** [comment]
I have the same problem. I can't get any color when running nano in powershell (`pwsh.exe`), no matter what I do. It runs fine, but no color.

Can someone please provide:
1. a functioning `.nanorc` and the location of that?
2. the correct `$env:path` and location?
3. Do we also need to set our own `$HOME`?
4. Do we need to manually enable `ENABLE_VIRTUAL_TERMINAL_PROCESSING`?  
   (New windows should handle this automatically, but may be need to enable manually for legacy code?)

**Comment by eabase** [comment]
You also said in your latest commit that you enabled `UTF-8`, yet it was compiled with it disabled. (See #35 ).

**Comment by akuropka** [comment]
I used the default `.nanorc` file included in the package, commented out the line following the `## To include most of the existing syntax definitions, you can do:` and adjusted the path to `c:/installfolder/whatever/*.nanorc`.
You can load the .nanorc file with the option `--rcfile=c:/installfolder/.nanorc`.
You can pack this into a one-liner batch file like `@"c:\Program Files\Nano\pkg_x86_64-w64-mingw32\bin\nano.exe" --rcfile="c:/Program Files/Nano/.nanorc" %*` to avoid typing the option everytime.

**Comment by eabase** [comment]
I just installed nano (both same & latest versions) on a new Window 10 Pro. I copied all the nano *config, environment* and al the various terminal and character coding settings I could think of to the new machine, but no matter what I do, I am not able to get colors showing in pwsh (core), even if they're working perfectly for all other programs I use via terminal. I have no idea what is causing this not to work ion this machine.

**Comment by rkitover** [comment]
By the way, for a nicer set of syntax highlighting definitions see:

https://github.com/scopatz/nanorc
